perses/annihilation/relative.py

```diff
@@ -1255,61 +1279,69 @@ def handle_periodic_torsion_force(self):
             #_logger.debug(f"\t\thandle_harmonic_torsion_forces: old_torsion parameters: {torsion_parameters}")
 
 
-            #get the indices in the hybrid system
+            # Get the indices in the hybrid system
             hybrid_index_list = [self._old_to_hybrid_map[old_index] for old_index in torsion_parameters[:4]]
             hybrid_index_set = set(hybrid_index_list)
 
-            #if all atoms are in the core, we'll need to find the corresponding parameters in the old system and
-            #interpolate
+            # If all atoms are in the core, we'll need to find the corresponding parameters in the old system and
+            # interpolate
             if hybrid_index_set.intersection(self._atom_classes['unique_old_atoms']) != set():
-                #then it goes to a standard force...
-                self._hybrid_system_forces['unique_atom_torsion_force'].addTorsion(hybrid_index_list[0], hybrid_index_list[1],
-                                                                        hybrid_index_list[2], hybrid_index_list[3], torsion_parameters[4],
-                                                                        torsion_parameters[5], torsion_parameters[6])
+                if self._flatten_torsions:
+                    force_params = [torsion_parameters[4], torsion_parameters[5], torsion_parameters[6], torsion_parameters[4], torsion_parameters[5], 0.]
+                    self._hybrid_system_forces['custom_torsion_force'].addTorsion(hybrid_index_list[0], hybrid_index_list[1], hybrid_index_list[2], hybrid_index_list[3], force_params)
+                else:
+                    # Then it goes to a standard force...
+                    self._hybrid_system_forces['unique_atom_torsion_force'].addTorsion(hybrid_index_list[0], hybrid_index_list[1],
+                                                                            hybrid_index_list[2], hybrid_index_list[3], torsion_parameters[4],
+                                                                            torsion_parameters[5], torsion_parameters[6])
```

Comment:
if we are flattening torsions, they have to go into a custom force so they can be turned on